packages/storefront-automation/tests/home_test.js

```diff
@@ -0,0 +1,12 @@
+Feature('home');
+
+Scenario('Load a product starting from the home page', (I) => {
+    I.amOnPage('/');
+    I.see('Summer Look');
+    I.see('Popular Catalogs');
+    I.click('Dresses');
+    I.see('Floral Dress');
+    I.click('Floral Dress');
+    I.amOnPage('/product/25592581M');
```

Comment:
Why do you have to go to the product page directly. `I.click` should take the browser to the product page.

Reply:
This is a WIP